src/Acheve.TestHost/IncludeContentAsUrl.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+
+namespace Microsoft.AspNetCore.TestHost
+{
+    /// <summary>
+    /// An implementation of <see cref="RequestContentOptions"/> that includes 
+    /// the [FromFornm] parameter as <see cref="IDictionary<>"/>
+    /// </summary>
+    public class IncludeContentAsUrl : RequestContentOptions
+    {
+        /// <inheritdoc/>
+        public override bool IncludeFromBodyAsContent => false;
+        /// <inheritdoc/>
+        public override bool IncludeFromFormAsContent => true;
+
+        /// <inheritdoc/>
+        public override Func<object, HttpContent> ContentBuilder =>
+            content => new FormUrlEncodedContent(ToKeyValue(content));
+
+        private IDictionary<string, string> ToKeyValue(object metaToken)
```

Comment:
I´m not a big fan of this implementation using recursion, but it can work.
Please, add some unit test to this class validating the behavior of the ContentBuilder method using different objects with different shapes. Use a Dictionary<string, string>, an object with nested objects as properties at least

Reply:
Fixed